src/atom_finder/util.clj

```diff
@@ -496,6 +496,36 @@
 
        (println "===================================================")))))
 
+(defn get-precedence-level
+  "Returns the precedence level of the IASTnode, returns nil otherwise"
+  [node]
+  (let [precedence-list
+        {;"currently supports binary operators(not including sizeof), comma operator and conditional operator"
+         IASTBinaryExpression/op_modulo 5 IASTBinaryExpression/op_multiply 5 IASTBinaryExpression/op_divide 5
```

Comment:
Why aren't all the precedence levels defined in this function? This should definitely have everything from 1-15 or else it won't really be useful outside your code.